test/test_scanner.py

```diff
@@ -261,6 +261,19 @@ def _file_test(self, url, filename, package, version):
                     "CVE-2018-1152"
                 ],
             ),
+            (
```

Comment:
Please group this test so it sits next to the other expat test in the array.

Reply:
Terri, I have grouped the tests together.